src/interfaces/chain.cpp

```diff
@@ -187,11 +187,11 @@ class NotificationsHandlerImpl : public Handler, CValidationInterface
         const CBlockIndex* index,
         const std::vector<CTransactionRef>& tx_conflicted) override
     {
-        m_notifications->BlockConnected(*block, tx_conflicted);
```

Comment:
should drop `getBlockDepth` no? https://github.com/bitcoin/bitcoin/pull/15931/files#diff-efaf73429006d059e7fffab1bf2879769c90187f583d4857b3275b7bf79368d1L61-L66

Reply:
Also, is there a reason we don't currently have waitForNotificationsIfNewBlocksConnected? https://github.com/bitcoin/bitcoin/pull/15931/files#diff-efaf73429006d059e7fffab1bf2879769c90187f583d4857b3275b7bf79368d1L356-L362